### ISSUE TITLE
Mechanical Attachments / slight MANGA SPOILERS

### ISSUE THREAD

**Issue by getvood** [MineInAbyss/Issue-Tracker#145]
### **MANGA SPOILERS**
In chapter 65, they show that limbs can be, for the most part replaced by mechanical parts called "works. I always thought that the mobs in the abyss are too weak, especially since getting diamonds and a bow is all you really need to kill the strongest custom mob in the game currently, being the Orb Piercer, while in the anime even moon whistles who are extremely experienced delvers are almost certain to die when confronting an Orb Piercer, and it's not as simple as raising it's health and damage because in that case, not even the most hardened players would be able to kill it head-on without an elaborate plan or cheesing the fight.

The issue is that there's almost no difference in armor and weapon between players who have grinded a little vs players who grinded a substantial amount more and I think this gap can be created with the introduction of Mechanical Attachments. I think this change would be really good because it,
- Introduces a New Tier of gameplay
- Encourages people to play more (and explore more depending on the implementation)
- Enables deadly creatures to become even more deadly

### Describe the solution you'd like.

tldr;
- 4 total uses
- -5% mining speed per use
- -1.5 hearts/ use _(More susceptible to curse dmg)_
- +15% speed/ use
- +15% strength/ use
- +1.5 armor stat/ use
- -10% hunger depletion/ use
- negative effects linger for a period of time after removing attachments
- not an armor slot attachment _(attachments and armor can be worn at the same time)_

I'll list the features first, then I'll address some concerns and problems in a separate paragraph. I don't like the idea that every relic and attachment that comes a player's way should be 100% useful, and it also reflects well on Made In Abyss that alot of the stronger, more useful relics come with some downsides and they have to think carefully about whether or not it's useful to them.

And with that, the Mechanical Attachments (Works) would be an upgrade to the player in a slight stat increase in almost everything, with a decrease in dexterity which shows as Mining fatigue, but the main change is that for every attachment that is used, the player loses 1.5 hearts permanently, which is replaced by a permanent armor stat that is present even if no armor is being worn, this can be done a total of 4 times, which represents the player's 4 limbs. With this the player takes significantly less damage to all types of damage except the curse which they are now highly more susceptible to.

As for concerns, there's only 3 that I thought of, 

- Programming. I'm not familiar with it but I'm assuming it's hard to implement so this idea will be on the backburner if accepted. 
- Replacing Limbs. There's a discrepancy between works being a permanent thing in Made in Abyss and somehow not being a permanent thing in Mine in Abyss, A good balance could be that after placing them on, you will still have the downsides of the works for a period of time afterwards.
- Durability. In my opinion, I think instead of the item breaking when broken, it would be better to be able to repair it with parts such as Iron Ingots for example. And on top of that, not being able to place mending on it, so a relative flow of resources is needed to keep it functional.

### Describe alternatives you've considered.

The main point of this is to add a new tier of gameplay which would require more commitment and enough grinding to have it be separate enough to have it be considered "Endgame" gear. This can be achieved in an infinite number of ways, this is just the most straight forward, Canon to Made in Abyss, and has complexity.

### Agreements

- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have ensured the feature I'm requesting isn't already in the latest supported Paper build.

### Other

Not a concrete Idea, just a general suggestion about some kindof of Semi-Permanent Late-game upgrade. So anything that anyone would like to suggest in terms of upgrades is appreciated.

The values I put in the tldr section we're just off the top of my head and that looked right, revision is almost definitely needed just to check if the player won't become near invincible or invincible with all 4 attachments and the best armor.

How to obtaining the relic will be for future discussion because there'll be more ways to obtain relics in the future that are in my opinion, more appropriate for this kind of relic.
